beluga/include/beluga/actions/overlay.hpp

```diff
@@ -30,96 +30,90 @@ namespace beluga::actions {
 
 namespace detail {
```

Comment:
nit: GitHub doesn't let me comment on the exact line, but I think this header is not needed here:
```cpp
#include <range/v3/view/transform.hpp>
```